test/lint/golangci.sh

```diff
@@ -29,6 +26,11 @@ if [ -z "${target_branch}" ]; then
   return
 fi
 
+if ! git show-ref --quiet "refs/heads/${target_branch}" --quiet >/dev/null 2>&1; then
+    # Attempt to fetch the reference (Github uses a shallow clone).
+    git fetch -q origin "${target_branch}" || true
```

Comment:
should we still have the `|| true` bit here?

Reply:
Probably not! :facepalm: